functional-tests/src/test/java/com/redhat/che/functional/tests/BayesianPomXml.java

```diff
@@ -44,14 +43,16 @@
 
   @BeforeClass
   public void setUp() throws Exception {
+    if (cheHost.equals(CHE_PROD_PREVIEW_URL)) {
```

Comment:
@Katka92 would be nice to add links to the issues there in order to clarify why the tests are skipped